### ISSUE TITLE
Stuck on "Sending" doesn't go into the blockchain. "Transaction Not Found" on bitpay

### ISSUE THREAD

**Issue by Xue123123123** [bitpay/copay#7325]
help me please

**Comment by Xue123123123** [comment]
Do i just resend funds or wait it out as i need it to show up asap

**Comment by kater19** [comment]
I've been meaning to post for a couple weeks now here as I have the same issue. The first transaction I tried sending on November 12th. Now, I have 4 transactions that are "sending" and not yet broadcast to the network. The amounts have been deducted from my balance and the transactions usually disappear and I have to scan addresses and then clear the transaction cache for them to show on the recent list temporarily.

It's been 18 days since the first stuck transaction and now I'm concerned in not seeing these funds again although everyone keeps telling me the btc is still in my wallet if not shown on blockchain.

**Comment by JDonadio** [comment]
Should be working now. Sorry about the problems. If you cannot find your transaction on insight, you can always search it in any other block explorer.

**Comment by kater19** [comment]
Why is this issue closed? It's not working. Now even after rescanning and clearing cache, the transactions are not showing up. The money is still gone though!

**Comment by JDonadio** [comment]
What platform are you using ?

**Comment by kater19** [comment]
I'm on my mobile bitpay wallet.


On Dec 1, 2017 at 12:18 PM, <Javier Donadío <notifications@github.com>>
wrote:

What platform are you using ?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/bitpay/copay/issues/7325#issuecomment-348568590>, or mute
the thread

**Comment by JDonadio** [comment]
Tap on view on blockchain button, when the browser open the URL select it and copy it before all the page did load. And paste the link here

**Comment by kater19** [comment]
Well for one, I just said that I cannot even see the transactions anyomore in my wallet so there's nothing to click on. I found a confirmation email from one of the 4 and copied the link from there:
https://insight.bitpay.com/tx/f83f121c451fff2a31e9f757877f43330080c2da3fed8db24cf5fe675afc6aaf

Still no transaction found

**Comment by JDonadio** [comment]
Your transaction doesn't exist. Has been dropped from the mempool

https://blockchain.info/search?search=f83f121c451fff2a31e9f757877f43330080c2da3fed8db24cf5fe675afc6aaf

**Comment by kater19** [comment]
Well then where is my bitcoin? Attached is a screenshot from yesterday of
the 4 transactions that have been sending - none of them have ever showed
up on the blockchain - oldest one is now 19 days old.


On Dec 1, 2017 at 12:30 PM, <Javier Donadío <notifications@github.com>>
wrote:

Your transaction doesn't exist. Has been dropped from the mempool

https://blockchain.info/search?search=f83f121c451fff2a31e9f757877f43330080c2da3fed8db24cf5fe675afc6aaf

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/bitpay/copay/issues/7325#issuecomment-348571700>, or mute
the thread

**Comment by JDonadio** [comment]
If you send transactinos and they are not confirmed, you are using unconfirmed funds for each one. So some of these transactions (or even all) could be dropped from the mempool and the other ones will not be either broadcasted.

**Comment by kater19** [comment]
Okay, then where is my bitcoin Now? If they were never broadcasted or were
dropped why are the funds still gone from my wallet 19 days later?


On Dec 1, 2017 at 12:36 PM, <Javier Donadío <notifications@github.com>>
wrote:

If you send transactinos and they are not confirmed, you are using
unconfirmed funds for each one. So some of these transactions (or even all)
could be dropped from the mempool and the other ones will not be either
broadcasted.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/bitpay/copay/issues/7325#issuecomment-348573275>, or mute
the thread

**Comment by MrSeanBon** [comment]
This exact thing has happened to me! It’s been 3 days now. I followed the advice given here and cleared cache. Now the “sending” transaction doesn’t even show in my history at all! Funds are gone, never hit the Blockchain, and no history in app to show now... 
I am extremely nervous now and don’t trust Copay to do any transactions.

I did NOT do NOT have spend unconfirmed enabled...
What can we do? I’ve been waiting for the transaction to either show up on the blockchain or drop completely and go back in my wallet... neither has happened for days

**Comment by kater19** [comment]
Why is this still marked as closed. I have 5 transactions now where funds have been taken out of my wallet, do not exist on blockchain, and have not been returned to me. It's been almost a month. This is a serious issue and from what I understand, if the transaction is not okn blockchain then btc must be in wallet. That is why I am now here in the support forum for my wallet asking that something be done to fix this!

**Comment by omietrice** [comment]
This support is garbage. They are prentending like it's "Bitcoin" but copay is either stealing funds or up to something shady.

**Comment by luckykcc17** [comment]
I am having same issue.....sent bitcoin with Economy fee which usually works fine but now it is stuck in _sending_ for about 8 hours now. When I search blockchain with sent address, no transaction shows up so this means that Bitpay has not even broadcasted the transaction to the network.  The bitcoin amount is gone from my wallet, the transaction doesn't even show up in my history...I have to "scan for funds" in order to have it appear but it disappears again if I close Bitpay. THIS IS NOT A BITCOIN NETWORK ISSUE!!! THIS IS A BITPAY ISSUE!!!

**Comment by luckykcc17** [comment]
I just want my bitcoin back in my wallet, and since Bitpay has it....Bitpay needs to fix this immediately and either send the transaction or give me back my bitcoins. Otherwise, this would be defined as stealing!!!!

**Comment by kater19** [comment]
I have reached out to bitpay/copay support regarding this issue on 8 different occasions since my last post in this thread. I have 7 transactions in my wallet that are currently in bitpay limbo now. Oldest one of the 7 was sent on November 12. 

Here is something else though...one of these transactions I did as a test run and sent funds between 2 copay wallets within the same app. The transaction shows as "sending" from one wallet and shows up in the other wallet as "receiving" yet the transaction doesn't exist on the blockchain. INTERESTING.

Another instance I am pretty pissed about as well...I tried using the copay recovery tool, and to my surprise the tool recovered some of my bitcoin that was in limbo...yay! NOPE. Sent those "recovered" funds only to then once again find that the transaction doesn't exist on the blockchain. I don't have record of this transaction other than a couple screenshots as it wasn't done from my wallet but the recovery tool website. So as I cannot pull up any important information regarding that transaction, at this point I am assuming they are gone forever.

Oh but the worst part of this whole 2 month long shitshow that's been going on in my copay and bitpay wallets is that THE SUPPORT DOES NOT REPLY!!!!! I have posted in different threads, sent email to support, posted issue in general bitpay support. I have screenshots, email confirmations, wallet info, btc addresses, transaction id's, dates, amounts, YET NO ONE WILL EVEN FUCKING REPLY TO ME.
